db/seeds.rb

```diff
@@ -1,7 +1,32 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+include Fake
```

Comment:
Although it was taught in the foundation, try to avoid using `include SomeModuleName` in the global scope, it will add the module methods to all objects.

```ruby
# Avoid:
# Top of file, in global scope, not within a class definition
include SomeModuleName 

# -vs-

# Good, not within global scope:
class SomethingOrOther
  include SomeModuleName
  ...
end
```

The reason for avoiding `include Fake` here is it is making all of the `Fake` methods available in global scope and on all objects. This is over-exposing those methods and cluttering the global scope and the `Object` class, potentially overriding valid methods that happen to share the same name but have different behaviour.

To use `Fake`, remove `include Fake`, and change the `Fake` methods to be class methods by prefixing the methods with `self.`:

```diff
module Fake

- def some_method_name
+ def self.some_method_name
```

`Fake.some_method_name` ought to be callable from within `seeds.rb` after the above changes.